tools/extend_entry_times.py

```diff
@@ -3,22 +3,22 @@
 import os
 import tqdm
 
-pre_extend_times = [250, 254, 255, 256, 257]
-post_extend_time = 258
+# pre_extend_times = [250, 254, 255, 256, 257]
```

Comment:
I use this script every time I update the atlas, so all the commented-out lines should be added back.